wireapp/kalium
# fix: order of devices and conversation list [AR-2009] [AR-2070]

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Lists of devices after login and conversations after fresh install look like they are arranged randomly.

### Solutions

Sort list of devices by reverse chronological order and conversations by reverse chronological, and if that's enough or there is no "updated time" then by alphabetical order.

### Testing

#### Test Coverage (Optional)


#### How to Test

Install fresh version to see the conversation list order, have too many registered devices when logging in to see devices list order.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
